packages/relay/src/lib/eth.ts

```diff
@@ -507,25 +507,18 @@ export class EthImpl implements Eth {
 
```

Comment:
10 lines above this, there is a comment, which I think can be removed now with this PR.

Reply:
Thanks